source/manual/deploy-banner-to-certain-pages.html.md

```diff
@@ -0,0 +1,51 @@
+---
+owner_slack: "#govuk-2ndline-tech"
+title: Deploy a banner to certain pages
+parent: "/manual.html"
+layout: manual_layout
+section: Frontend
+important: true
+---
+
+This documentation should be used when we receive a request to publish a banner
+to a single, or group of pages. If you want to publish a sitewide banner then
+reference [deploy emergency banner][emergency-banner] or [deploy non-emergency
+banner][non-emergency-banner].
+
+We can publish a few different banner types, depending on the ask. For example,
+we've previously published a [notice banner][notice-banner] for a
+mini-reshuffle and an [intervention banner][intervention-banner] for a research
+panel.
+
+The [component guide][component-guide] has a list of GOV.UK components which
+includes banners like ([notice][notice] and [intervention][intervention] and
+their variants, e.g [notice with markup in title][notice-markup].
+
+## Prerequisites
+
+* banner text
+
+* banner location on page/s
+
+## Steps
+
+1. Find the rendering app of the chosen pages (visit www.gov.uk/api/content/base-path and look for `rendering_app`)
+1. Create a branch introducing the component in the relevant view and with a helper method that only displays the banner for certain base_paths (e.g [org pages in Collections][notice-banner-files]).
+1. Preview the banner locally (e.g using docker [collections-app-live][collections-live], or deploy a branch to integration)
+1. Check the positioning of the banner and adjust accordingly. See [an example][notice-margin] of adjusting the margin of the notice banner.
```

Comment:
It would be good to add one last action:

```suggestion
1. Check the positioning of the banner and adjust accordingly. See [an example][notice-margin] of adjusting the margin of the notice banner.

Once your changes are merged, you should also [write a technical debt card](https://trello.com/b/oPnw6v3r/govuk-tech-debt) for the banner you've added, as this is a non-standard piece of technical work that has been essentially 'hacked in' due to limitations in our current platform.
```

Please also write a card for the banner you added last week! 😄

Reply:
Haha yes, I'll do that now. Have also added your suggestion 💯